src/cli.import.job.ts

```diff
@@ -0,0 +1,93 @@
+import { Bounds, TileMatrixSets } from '@basemaps/geo';
+import { Config, extractYearRangeFromName, LogConfig, S3FsJson } from '@basemaps/shared';
+import { Command, flags } from '@oclif/command';
+import { PrettyTransform } from 'pretty-json-log';
+import * as z from 'zod';
+import { ConfigImagerySchema, zImageConfig, zNamedBounds } from './imagery.config';
+const zJob = z.object({
+  id: z.string(),
+  name: z.string(),
+  output: z.object({
+    tileMatrix: z.string(),
+    files: z.array(zNamedBounds),
+  }),
+});
+
+const ResolutionRegex = /((?:\d[\.\-])?\d+)m/;
```

Comment:
Does the regex able to parse the the name with `RGBA` or `RGB` at the end?

Reply:
This is the problem with magic regexp, this one I believe will just look for `[0-9]m` or `[0-9].[0-9]*m` eg `1m` or `0.05m` in the title so it shouldnt matter where it happens in the string